cmd/mock-registry/mock_registry.go

```diff
@@ -29,15 +29,15 @@ func main() {
 
 	apps := LoadAnsibleApps(args.AppFile)
 
-	http.HandleFunc("/ansibleapps", handler(args, apps, GetAnsibleApps))
+	http.HandleFunc("/apbs", handler(args, apps, GetAnsibleApps))
```

Comment:
Not sure if apbs is a good url. `/playbookbundles` , `/bundles`. And Looks like I missed one, I didn't change `GetAnsibleApp`.